components/emoji_picker/README.md

```diff
@@ -0,0 +1,168 @@
+We don't support all the emojis, like those which have multiple skin tone modifier.
+We also dont support the regional / modifier and emojis-skin-tones set of emojis.
+Since our component doesn't handle all the emojis, we need to extract the emojis from the json file.
+
+
+To accomplish the task, and get only the necessary and useful emojis we need to do the following steps:
```

Comment:
We would need to do this whenever the emoji json is updated right?

Reply:
Probably yes... because we dont know how it will be the updated version.